src/static/helpers/slasHelper.ts

```diff
@@ -188,6 +252,29 @@ export async function loginGuestUser(
   return slasClient.getAccessToken({body: tokenBody});
 }
 
+export async function loginGuestUserPrivateClient(
+  slasClient: ShopperLogin<{
+    shortCode: string;
+    organizationId: string;
+    clientId: string;
+    siteId: string;
+  }>,
+  parameters: {
+    redirectURI: string;
+    refreshToken?: string;
+    codeVerifier?: string;
+    code?: string;
+    usid?: string;
+  },
+  credentials?: {
+    client_id: string;
+    client_secret: string;
+  }
+  ): Promise<TokenResponse> {
+
+  return await getAccessToken(slasClient, parameters, credentials);
+}
+
```

Comment:
commerce-sdk-isomorphic assumes we are using a SLAS public client

When using a SLAS private client, we want to skip the call to `/authorize`. We can immediately call the `/token` endpoint